encodings/fsst/Cargo.toml

```diff
@@ -38,3 +42,8 @@ vortex-array = { workspace = true, features = ["test-harness"] }
 [[bench]]
 name = "fsst_compress"
 harness = false
+
+[[bench]]
+name = "chunked_dict_array_builder"
```

Comment:
I should split this into two benches, and move the non fsst dict builder benchmarks to vortex-array